css-box-model/styles.css

```diff
@@ -0,0 +1,9 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin: 0 12px;
```

Comment:
The spec doesn't specify what the top and bottom margin should be, so please don't manually set them:

---

![image](https://user-images.githubusercontent.com/16143771/176503849-4377d037-8424-49cb-8f6a-4ac130fc9efa.png)